backend/src/main/java/io/debezium/configserver/model/AdditionalPropertyMetadata.java

```diff
@@ -0,0 +1,21 @@
+package io.debezium.configserver.model;
+
+import java.util.List;
+
+public class AdditionalPropertyMetadata {
+
+    public boolean isMandatory;
```

Comment:
Can these all be made final?

Reply:
Oh, sure! I just continued your work though. My own head was sleepy on that one while coding.
I often only see such things on reviews when being the reviewer and don't ask myself that question often enough while coding. :pray:

Reply:
Done